### DISCUSSION TITLE
Is there a way to restrict access based on azure ad group when logged in?

### DISCUSSION THREAD

**Discussion by vSivarajah** [provectus/kafka-ui#3494] [Q&A] [answered]
Hi, i managed to succesfully setup SSO through Azure but how can i restrict a user from logging in if they are not part of a particular AD group?

This is how i configured SSO for Azure:

```
AUTH_TYPE: OAUTH2
AUTH_OAUTH2_CLIENT_AUTH0_CLIENTID: {client_id}
AUTH_OAUTH2_CLIENT_AUTH0_CLIENTSECRET: {client_secret}
AUTH_OAUTH2_CLIENT_AUTH0_ISSUER-URI: "https://login.microsoftonline.com/{tenant_id}/v2.0"
AUTH_OAUTH2_CLIENT_AUTH0_provider: auth0
AUTH_OAUTH2_CLIENT_AUTH0_SCOPE: openid
```

**Comment by Haarolean** [comment, marked as answer]
Hi, it'd be available with RBAC after we're done with #2844

**Reply by vSivarajah** [reply]
Thank you for the quick response, looking forward for the change!
